.github/workflows/nightly_run.yml

```diff
@@ -39,24 +47,16 @@ jobs:
     with:
       os: "ubuntu-20.04"
       arch: "X86"
-
-  build_llvm_libraries_on_ubuntu_2204:
-    uses: ./.github/workflows/build_llvm_libraries.yml
-    with:
-      os: "ubuntu-22.04"
-      arch: "X86"
   
   build_wamrc:
     needs:
-      [build_llvm_libraries_on_ubuntu_2004, build_llvm_libraries_on_ubuntu_2204]
+      [build_llvm_libraries_on_ubuntu_2004]
```

Comment:
Some of the tests on ubuntu-22.04 are removed, are they already tested in the compilation_on_android_ubuntu.yml?

Reply:
can you elaborate a bit more?

They're used to be the same workflow (nightly one and per-PR one) but nightly had sanitizers tests. 

So for me the only thing that we lose if sanitizers tests on ubuntu 22, which are unlikely to fail if they pass for ubuntu 20

Reply:
Got it, it is good to me.